partiql-types/src/main/kotlin/org/partiql/value/datetime/impl/SqlDate.kt

```diff
@@ -33,8 +33,8 @@ internal data class SqlDate private constructor(
         @JvmStatic
         @Throws(DateTimeException::class)
         fun of(year: Int, month: Int, day: Int): Date {
-            if (year < 1 || year > 9999)
-                throw DateTimeException("Expect Year Field to be between 1 to 9999, but received $year")
+            if (year < 0 || year > 9999)
+                throw DateTimeException("Expect Year Field to be between 0 to 9999, but received $year")
```

Comment:
Is this change expected? If so:

See [Ion Spec](https://amazon-ion.github.io/ion-docs/docs/spec.html#timestamp)
Also, SQL-1999 Section 6.1 Table 11